src/chigame/users/models.py

```diff
@@ -22,7 +31,9 @@ class User(AbstractUser):
     first_name = None  # type: ignore
     last_name = None  # type: ignore
     email = models.EmailField(_("email address"), unique=True)
-    username = None  # type: ignore
+    username = models.CharField(
```

Comment:
Especially if you're adding a new field to a model you need to update the team and the whole project once you've got approval, since there could be many things that rely on the current set up of the `User` model that changing this would impact.

Only upon getting clearance from both the users team and after asking everyone in the general slack channel will I merge these changes.

Reply:
<img width="733" alt="Screenshot 2023-12-01 at 12 28 03 PM" src="https://github.com/uchicago-cs/chigame/assets/107411138/fdab042e-c0f4-403b-a8dc-4dd1ca3b4d4b">

Reply:
got clearance from the `users` team! will ask in the general slack channel

Reply:
got a thumbs up in the general channel!!